msg/SpawnPose.msg

```diff
@@ -0,0 +1,6 @@
+# A named spawn pose (point)
+
+string point_name                         # Name of a spawn point.
```

Comment:
Good question, I see two options:
- It is unique
- It is not unique. This would mean that for a single name there can be several poses. Since we spawn by a pose anyway, this makes sense (e.g. allows the sim to communicate several alternatives that are all valid). 
I would go with the second option

Reply:
Do you know what are the spawn point name requirements in the other simulators?

Reply:
None to my knowledge.

Reply:
I think this makes sense as this is something originating from the simulator - a part of creation of a virtual scene.